src/Core/Injector/InjectionCreator.php

```diff
@@ -9,6 +9,9 @@ class InjectionCreator implements Factory
 {
     public function create($class, array $params = [])
```

Comment:
```suggestion
    public function create($classOrMethod, array $params = [])
```

Also update references to `$class` elsewhere in this method